azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ChatContainer.kt

```diff
@@ -88,12 +89,15 @@ internal class ChatContainer(
 
             addTypedBuilder { remoteOptions }
 
+            addTypedBuilder { ChatEventHandler() }
```

Comment:
Just pass ChatEventHandler into the ChatService, no need to provide it

Reply:
I think it is good to follow constructor injection. We will create factory in future if more object needs to be injected in service layer.